lib/run.js

```diff
@@ -216,6 +219,13 @@ function run (opts, tracker, cb) {
       tracker.emit('response', this, statusCode, resBytes, responseTime)
       const codeIndex = Math.floor(parseInt(statusCode) / 100) - 1
       statusCodes[codeIndex] += 1
+
+      if (!statusCodeStats[statusCode]) {
+        statusCodeStats[statusCode] = { count: 1 }
+      } else {
+        statusCodeStats[statusCode].count++
+      }
```

Comment:
is there any reason to not use the object values as the count, instead of having an object with a single property `count`? It would be more consistent with how the other stats are aggregated

Reply:
@simoneb , you are absolutely right, there are many ways this could be improved. I was not certain of how far I can mod the code and still remain within the original author's intent. My goal was `additive` change only without impacting any of the existing functionality. If you believe that you can take this further, please do.

Reply:
As far as `object with a  single property`, I envision that there may be other attributes aggregated about each individual statusCode, other than a count, and thus wanted to provide a harness for that. An example could be request timing average per code.

Reply:
@maxfortun on the overall approach let's hear what @mcollina thinks. On the count property, if it's not necessary, I'd say avoid it